packages/meson.rb

```diff
@@ -3,26 +3,24 @@
 class Meson < Package
   description 'The Meson Build System'
   homepage 'http://mesonbuild.com/'
-  @_ver = '0.57.1'
+  @_ver = '0.57.2'
   version @_ver
   license 'Apache-2.0'
   compatibility 'all'
   source_url "https://github.com/mesonbuild/meson/archive/#{@_ver}.tar.gz"
-  source_sha256 '0c043c9b5350e9087cd4f6becf6c0d10b1d618ca3f919e0dcca2cdf342360d5d'
-
-  depends_on 'ninja'
```

Comment:
Why was this dependency removed?  Don't we still need ninja with meson?

Reply:
At the time this was built, ninja was still part of `core_packages.txt`, so my scripts dutifully removed the dependency!

I'll put it back in...

Reply:
Adjust your script to ignore any dependencies from packages in `core_packages.txt`. :)